Flow.Launcher.Plugin/AllowedLanguage.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace Flow.Launcher.Plugin
 {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
why disable the warning here? If you would like to not include this in this pr, just leave it here and address it in the future.

Reply:
Seemed like a fairly self documenting class - but you're right. I'll fix it.